### DISCUSSION TITLE
Support for reverse proxying via sub folder

### DISCUSSION THREAD

**Discussion by englut** [immich-app/immich#12482] [Q&A] [answered]
I'm trying to avoid exposing my immich instance to being discover-able as much as possible (especially via DNS records). And was wondering if it's possible to reverse proxy via sub folder?

**Comment by mmomjian** [comment]
Hi, this won’t be added any time soon

just use a wildcard SSL cert. DNS records generally can’t be scraped/queried if not already known. 

#1679

**Reply by englut** [reply]
Is there any particular reason why?

**Reply by mmomjian** [reply]
It’s not trivial to add to the various libraries that we use, and there’s little benefit to justify the amount of added work.

**Reply by englut** [reply]
I appreciate that reasoning. I can understand how it does seem to be a specific use-case, if the code base isn't designed with relative paths in mind. Is the immich team open to a PR to support this?

Also, DNS records are easily discoverable:
https://www.shodan.io/domain/immich.app

**Reply by mmomjian** [reply, marked as answer]
I’m not certain that a PR would be accepted because it’s likely to add a lot of complexity. You may want to ask on the discord in #contributing before working on it. 

That website is probably using SSL cert records. I just tested with my domain and none of the subdomains are visible, because I use a wildcard SSL cert.

**Reply by englut** [reply]
Okay. Thanks I’ll try both!
